pkgs/applications/audio/stargate/default.nix

```
@@ -0,0 +1,167 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, substituteAll
+, alsa-lib
+, autoconf
+, automake
+, ffmpegSupport ? true
+, ffmpeg
+, fftw
+, fftwSinglePrec
+, file
+, gdb
+, gettext
+, jq
+, lameSupport ? true
+, lame
+, libsbsms
+, libsndfile
+, pkg-config
+, portaudio
+, portmidi
+, python3
+, python3Packages
```

```suggestion
, python3
```
Please use python3.pkgs to make sure we always use the same python.